client/.env

```diff
@@ -1 +1,2 @@
-REACT_APP_BASE_URL=http://localhost:4000
\ No newline at end of file
+REACT_APP_BASE_URL=http://localhost:4000
```

Comment:
can you add these config to .env.production file?
.env is for development let's not change that

Reply:
Added .env.production file for production